src/CommandingOfficers/Modifiers/COMovementModifier.java

```diff
@@ -1,12 +1,11 @@
 package CommandingOfficers.Modifiers;
 
+import java.io.Serializable;
 import java.util.ArrayList;
-
 import CommandingOfficers.Commander;
-import CommandingOfficers.Modifiers.COModifier.GenericCOModifier;
 import Units.UnitModel;
 
-public class COMovementModifier extends GenericCOModifier
+public class COMovementModifier implements Serializable
```

Comment:
Strikes me as a bit odd that this doesn't inherit from `UnitModifier`.

Reply:
Yeah. That was a temporary thing because I hadn't set up movement in the `UnitModifier` interface yet.